### PR TITLE
Add jupyter book build requirements

### PR DESCRIPTION
**Pull request by hemajv**
## Related Issues and Dependencies

#27

## This introduces a breaking change

- [ ] Yes
- [X] No

## Description

Adding requirements for Meteor pipeline to build JupyterBook

### CODE REVIEW

**Review by harshad16 — APPROVED** [review]
lgtm
thanks :100:

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+title: Data Driven Development
+author: AICoE
```
Comment (pacospace) [inline _config.yml:2]: Red Hat Artificial Intelligence Center of Excellence (AICoE)

File: _config.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+logo: logo.png
```
Comment (pacospace) [inline _config.yml:3]: where is this image located?

**Review by pacospace — COMMENTED** [review]
Two minor comments, otherwise great! 💯

File: _config.yml (hunk 2 above)
Comment (hemajv) [reply _config.yml:3]: ah good catch! will upload the image :+1:

**Review by pacospace — APPROVED** [review]
💯

**Pull request merged**